dali/pipeline/operators/reader/loader/video_loader.h

```diff
@@ -114,6 +114,7 @@ class VideoLoader : public Loader<GPUBackend, SequenceWrapper> {
   }
 
   void init() {
+    cuvidInit(0);
```

Comment:
Please add some error checking here and meaningful message about `that your env required video capability. If you are using docker -  refer to https://github.com/NVIDIA/nvidia-docker/wiki/Usage`, or something similar.

Reply:
Done